src/js/utils/queryUtils.js

```diff
@@ -0,0 +1,32 @@
+import { queryTypes } from '../constants/queryConstants';
+
+// cannot destructure enum as it is a keyword
+export const determineQueryType = ({
+  type,
+  is_range,
+  format,
+  ...otherProps
+}) => {
+  if (type === 'string') {
+    if (format === 'date') return queryTypes.DATE;
+    else if (otherProps.enum) return queryTypes.SELECT;
+    else return queryTypes.INPUT;
+  } else if (type === 'number') {
+    if (is_range) return queryTypes.RANGE;
+    else return queryTypes.NUMBER_INPUT;
+  } else return queryTypes.INPUT;
```

Comment:
FOr legibility, indent this statement or wrap it in curly braces to put it at the same level as its counterparts